container_pipeline/scanners/rpm_verify.py

```diff
@@ -80,6 +80,6 @@ def process_output(self, json_data):
         data = {}
         data["scanner_name"] = self.scanner_name
         # TODO: More verifcation and validation on the data
-        data["msg"] = "RPM verify results."
+        data["msg"] = json_data["Summary"]
```

Comment:
using different style in different scanners? is it by design?

Reply:
Different style in what sense? Can you elaborate?

Reply:
My thought was if we can have a single type of variable for summary. here we are using string and list. but I see the data['msg'] is having only string. I was not able to get the flow. thanks got it. and looks good to me.